tempesta_fw/ss_skb.c

```diff
@@ -698,7 +701,8 @@ __skb_fragment(struct sk_buff *skb, char *pspt, int len, TfwStr *it)
 	 */
 	if (len > 0) {
 		offset = pspt - (char *)skb_frag_address(&si->frags[0]);
-		if (unlikely(!offset && (len <= ss_skb_tailroom(skb)))) {
+		if (unlikely(!offset && (len <= ss_skb_tailroom(skb)))
+		    && !__ss_skb_tail_is_locked(skb)) {
```

Comment:
IIRC there is kernel code which manipulates with the tail pointer, so shouldn't we rather patch `ss_skb_put()` and `skb_put()`?